.github/workflows/cronJob.yaml

```diff
@@ -74,16 +74,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # Existing LTI release tags can be added to obtain build results.
+        tag: [ 'lsp4ij-market-0.0.2-integration' ] # Can specify tags or branches such as '24.0.6' and 'main'
```

Comment:
'tag' array is for specifying the LTI release tags. Can we run LTI builds on main branch even if 'tag' array is empty? 
rather than specifying 'main' inside the 'tag' array. Is there any way for that?

Reply:
Yes, if 'tag' array is empty , the default value `main `will run.